src/Playwright/Program.cs

```diff
@@ -37,6 +37,8 @@ public static int Main(string[] args)
             return p.Run(args);
         }
 
+        public static string GetExecutablePath() => Paths.GetExecutablePath();
```

Comment:
How am I suppose to get the path in a test should I implement test util? 
`playwright.ps1` is not working for me as I am on mac

Reply:
There is a `playwright.sh` next to it for your mac

Reply:
I wonder if you can try launching `dotnet Microsoft.Playwright.dll launch-server ...` as the process straight from the test. The DLL is compiled, and it'll be right there, and we can rely on `dotnet` existing because, well, we're a .NET app :-).

Reply:
I followed what other languages are doing -> https://github.com/microsoft/playwright-java/blob/a1ef49cd0389c6a32725a74ad01298069552ae0f/playwright/src/test/java/com/microsoft/playwright/TestBrowserTypeConnect.java#L68

& also in actual code, we are spawning from `.playwright/{os}` path. So ideally in test, we should be testing with the same path this ensures we are testing the correct functionality.

Reply:
Yes, if you run `dotnet Microsoft.Playwright.dll` it will spawn the OS specific version, using all the internal stuff

Reply:
Ideally, we shouldn’t be doing it in this way. Since we are not testing the actual functionality. In the source we are spawning on some different path & in the test we are spawning a different path. To test functionality should be the same.

Reply:
It is going to be the same, because we are going to be running the same code to compute it. And we don't need to expose it in the api that way.

Reply:
updated

Reply:
resolving this...